.mbedignore

```diff
@@ -1,8 +1,7 @@
-cryptoauthlib/app/
 cryptoauthlib/docs/
 cryptoauthlib/python/
 cryptoauthlib/third_party/
-cryptoauthlib/test/cmd-processor.c
+cryptoauthlib/test/*
```

Comment:
The commit message says "don't ignore `app`", but then the change also adds ignoring of the entire test folder. Was this intentional? If so, please make the change in a new commit and explain the rationale behind the commit in the commit message.